mssql/README.md

```diff
@@ -0,0 +1,38 @@
+# Spice Demo with SalesLT dataset in MSSQL
+
+This demo showcases how to use Spice with the SalesLT sample dataset in MSSQL.
+
+## Prerequisites
+
+- MSSQL Server with SalesLT sample dataset
+- Spice installed and configured
+
+## Steps
+1. Obtain your SQL server ADO connection string. It should look something like this:
+    `Server=tcp:server.database.windows.net,1433;Initial Catalog=nrc_health_testing;Persist Security Info=False;User ID={your_username};Password={your_password};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;`
+
+2. Copy `.env.example` as `.env` and put the connection string in the new `.env` file in the `MSSQL_CONNECTION_STRING` field
```

Comment:
Override with .env.local

Reply:
Can we just use `.env` and don't overcomplicate? I don't think there will be the case when someone would like to use both plus it can lead to potential issue when you updated a wrong `.env` file

Reply:
I've made it consistent with our other documentation/examples. So there's now `.env` which is the template and `.env.local` is ignored in the repo.